metrics/grafana/tikv_details.json

```diff
@@ -4532,7 +4532,6 @@
           },
           "yaxes": [
             {
-              "$$hashKey": "object:150",
```

Comment:
🤔where are these changes come from?

Reply:
I have no idea... ChatGPT told me this https://github.com/grafana/grafana/issues/11551 and it seems OK to just remove them